lib/studies/rally/routes.dart

```diff
@@ -0,0 +1,2 @@
+const String loginRoute = '/rally/login';
```

Comment:
Should the route consts for all the studies just be in one file instead of multiple?

Reply:
Torn on this one. I like that they are pretty independent. the demo types are breaking that right now, maybe those should be split too.